src/app/routes/cpsAsset/utils/getAssetType.js

```diff
@@ -1,5 +1,10 @@
-import path from 'ramda/src/path';
+import pathOr from 'ramda/src/pathOr';
 
-const getAssetType = pageData => path(['metadata', 'type'], pageData);
+const getCPSAssetType = pageData =>
+  pathOr(undefined, ['metadata', 'type'], pageData);
+const getOptimoAssetType = pageData =>
+  pathOr(undefined, ['data', 'article', 'metadata', 'type'], pageData);
```

Comment:
Probably best to stick with the default value of pathOr being `null` here, just to be consistent with the rest of the codebase